apps/studio/src/pages/home/uploadSong/PriceSummaryDialog.tsx

```diff
@@ -0,0 +1,240 @@
+import { FunctionComponent } from "react";
+import { useFormikContext } from "formik";
+import {
+  Dialog,
```

Comment:
Noticed you didn't use the elements>styled `Dialog.tsx`, is this another element we should transition out. Looks like the main difference is the styled background colors, which shows in between the delete account dialog and the price summary dialog.